docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx

```diff
@@ -0,0 +1,106 @@
+---
+ title: Managing Shared State in a Room with Session Metadata (Beta)
+ nav: 7.24
+---
+
+Session metadata is a shared state that is accessible to all participants in a room. It can be used for features such as pinning text or highlighting a particular participant. The session metadata is retained throughout the duration of a session and is cleared once all participants have left the room.
+
+## Differences between Session Metadata and Peer Metadata
+
+While [peer metadata](https://www.100ms.live/docs/android/v2/how--to-guides/interact-with-room/peer/peer-metadata-update) is associated with individual peers and each peer can have their own metadata, session metadata remains the same for every peer in the room.
+
+## How to store shared data with 100ms SDK
+
+To update the value of the session metadata for the current room session, call `setSessionMetadata` method on SDK instance. In the following example, the "test" string will be saved in the session metadata and accessible to all participants in the room as a shared state.
```

Comment:
is it `setSessionMetadata` or `setSessionMetaData` - seeing inconsistent casing between text and code snippet

Reply:
same for the getter method below